### ISSUE TITLE
1.7.10 EnderIO/Bees conflict

### ISSUE THREAD

**Issue by DrHysterion** [xJon/The-1.7.10-Pack#1962]
On the 1.7.10 pack's current and previous update versions the weather crystal required for the weather obelisk is uncraftable. It looks as though the intended recipe for creating the weather crystal was overridden in some form and makes the item unobtainable, and additionally shows the incorrect recipe.

**Comment by xJon** [comment]
Did this recipe work in the past? If so, on what version?

**Comment by DrHysterion** [comment]
It was functioning the last time I played the pack but that must have been 0.9 as it was likely a year ago. The recipe is supposed to be a diamond, two vibrant crystals and pulsating crystals however the pack seems to think it's made by a diamond in the crafting grid which gives diamond nuggets from translocator. I know it had worked previously because I saw people owned them on my server in the past.
